engine/src/test/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -213,5 +213,15 @@ class MaterializeWorkflowDescriptorActorSpec
 //      }
 //      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(customizedLocalBackendOptions).get)
     }
+
+    // TODO: PBE These tests have migrated here from WorkflowManagerActorSpec:
+    "Handle coercion failures gracefully" ignore {
```

Comment:
I took the ticket for MWDASpec already, so I might tidy this up into a "invalid coercions are invalid" test